src/components/MovieCard/MovieCard.scss

```diff
@@ -1 +1,3 @@
-// 
+.card-image img {
```

Comment:
Add a brief comment explaining the purpose of this CSS style for better readability.